en/docs/portal/transports/add-transport.md

```diff
@@ -8,27 +8,26 @@ To add a new transport, click the **Add New Transport** button at the top right
 
 | Fields               | Description   |
 | -------------------- | ------------- |
-| Transport Name       | Enter a name to identify this transport. If you omit this parameter, the HDT portal will generate a default name in the format of "Target Domain:Target Port|
+| Transport Name       | Enter a name to identify this transport. If you omit this parameter, HDT portal will generate a default name in "Target Domain:Target Port" format. |
 | Version Number       | Version number of the transport configuration in history. |
-| Application Protocol | <li><strong> HTTP </strong>: For HTTP protocol.</li> <li><strong> HTTPS </strong>: For HTTPS protocol with SNI (Server Name Indication). |
+| Application Protocol | <strong> HTTP </strong>: For HTTP protocol. <br> <strong> HTTPS </strong>: For HTTPS protocol with SNI (Server Name Indication). |
 | IP Version           | IP version supported by your application, IPv4, IPv6, or both. |
 
 ## Settings
 ![null](</docs/resources/images/transports/add-transport-settings-1.png>)
-![null](</docs/resources/images/transports/add-transport-settings-2.png>)
 
 | Fields               | Description   |
 | -------------------- | ------------- |
-| Fixed Shields        | The shield servers (the HDT servers that connection to your origin servers) will be fixed. |
-| Dedicated IP Service | The IP addresses for serving the transport is dedicated to the CNAME only. The option will NOT be applicable if you select HTTP or HTTPS as the Application Protocol. To use <strong> Dedicated IP Service </strong> for your HTTP/HTTPS applications, select Others as the Application Protocol.|
+| Fixed Shields        | If select Yes, the shield servers (HDT servers that connect to your origin servers) will be fixed, and you can choose the region of shield servers to be fixed in the drop-down list. |
```

Comment:
Please change "If select Yes" to "If you select Yes". I am not sure what "to be fixed" means. Do you mean "to display"?

Reply:
Yes, the "Fixed Shields" is like Burton said, a "static set of servers". Most of the cases, HDT will pick the best shield severs automatically based on detection result. While for some transports, the customers may hope the shied servers could be fixed/determinate due to security concerns. I was thinking “fixed” is a good adjective for this term, may I have your suggestions?

Reply:
The latest version:
| Fixed Shields | If you select Yes, the shield servers (HDT servers that connect to your origin servers) will be static, and you need to choose the region of shield servers in the drop-down list. |